brainrender_napari/brainrender_manager_widget.py

```diff
@@ -0,0 +1,54 @@
+"""
+A napari widget to download and update atlases.
+
+Available atlases are shown in a table view using the Qt model/view framework
+[Qt Model/View framework](https://doc.qt.io/qt-6/model-view-programming.html)
+"""
+
+from brainglobe_utils.qtpy.logo import header_widget
+from napari.viewer import Viewer
+from qtpy.QtWidgets import (
+    QGroupBox,
+    QVBoxLayout,
+    QWidget,
+)
+
+from brainrender_napari.widgets.atlas_manager_view import AtlasManagerView
+
+
+class BrainrenderManagerWidget(QWidget):
+    """The purpose of this class is
+    * to hold atlas visualisation widgets for napari
+    * coordinate between these widgets and napari
+    """
+
+    def __init__(self, napari_viewer: Viewer):
+        """Instantiates the atlas viewer widget
+        and sets up coordinating connections"""
+        super().__init__()
+
+        self._viewer = napari_viewer
+        self.setLayout(QVBoxLayout())
+        self.layout().addWidget(
+            header_widget(
+                "brainrender",
+                "Atlas management",
+                tutorial_file_name="manage-atlas-napari.html",
+                citation_doi="https://doi.org/10.7554/eLife.65751",
```

Comment:
Should this probably the the Atlas API citation?

Reply:
good point!